### ISSUE TITLE
Please add a possiblity to simply calculate costs by volume

### ISSUE THREAD

**Issue by Doprintityourself** [malnvenshorn/OctoPrint-CostEstimation#33]
Would apppreciate a simple cost calulation being added that just calculates by volume used

example There are 3 categories that have different price/volume:
- Price Categorie 1: Small and easy parts > 150mm^3 = 35cent/cm^3
- Price Categorie 2: bigger parts and more complex parts <200x200x200 = 50cent/cm^3
- Price Categorie 3: Big parts or very complex/difficult to print>200x200x200 =86cent/cm^3

So in summary you could choose between existing classic calculation  (one of 4 Checkmarks) or
one of the 3 above.

The calculation is than done with the specific section 
in this case would be only multiplying used filament cm^3* selected price Category 1,2,3 = Total costs €

Thank you very much in advance 
appreciate your work
